packages/calypso-polyfills/package.json

```diff
@@ -24,9 +24,9 @@
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
 	"dependencies": {
-		"core-js": "3.6.3",
-		"isomorphic-fetch": "2.2.1",
-		"regenerator-runtime": "0.13.3",
-		"svg4everybody": "2.1.9"
+		"core-js": "^3.6.3",
+		"isomorphic-fetch": "^2.2.1",
+		"regenerator-runtime": "^0.13.3",
+		"svg4everybody": "^2.1.9"
```

Comment:
Given this is a sharable package it makes sense that these dependencies are no longer pinned. Was this required for the PR or just some house keeping? Just checking that you didn't mean to add these to the root `package.json` too.

Reply:
I needed to make these ranges because of `regenerator-runtime`, which would have been duplicated. I went ahead and did the same for the other dependencies.